tox.ini

```diff
@@ -35,6 +36,15 @@ changedir = docs/source
 deps = -rdocs/requirements.txt
 commands = sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
+[testenv:py38-qa]
+basepython = python3.8
+deps =
+    black
+    isort
+commands =
+    black -t py36 --check --diff auditlog auditlog_tests setup.py runtests.py
+    isort --check-only --diff --profile black auditlog auditlog_tests setup.py runtests.py
```

Comment:
I think it would be good to add the isort profile as a setting generally to the project. This makes it easier when folk want to apply isort as they don't need to pass it in as a cli option.

Reply:
Thanks :+1: . I created a `project.tmol` file and move both `black` and `isort` configuration there.